### ISSUE TITLE
Display statistic graphs in the citing literature section

### ISSUE THREAD

**Issue by jcohenadad** [spinalcordtoolbox/spinalcordtoolbox#3857]
There is a way to embed googlesheet graphs as HTML's iframe, so I am wondering if we could do that in a sphinx-built page? If so, it would be neat to add a couple of visual statistics about the citing literature from [this source](https://docs.google.com/spreadsheets/d/1Utmf-btG7uXmu7RCBBx7fIKobm6yvXstX2Psc-UR2HA/edit#gid=1463629798).

Example of gsheet embedding [here](https://mcmedhacks.com/), with the HTML code for one example:

~~~
<iframe loading="lazy" width="600" height="371" seamless="" frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTMOgJ-8TyqT7IQGWDffe4krtXLgz8-9NO5d6HDr-coEHuvAlwOIfCjBl2fIpaP6iyWMQpyIk_lkJaz/pubchart?oid=1581143031&amp;format=interactive"></iframe>
~~~

**Comment by joshuacwnewton** [comment]
I've tried setting the "Publishing" setting on one of the graphs from the GSheet:

![image](https://user-images.githubusercontent.com/16181459/192843040-79ca26e5-6cc3-4801-ae7a-a814d80f1abd.png)

I then tried adding the embed iframe snippet to `studies.rst` using the `.. raw:: html` directive:

```
.. raw:: html

   <iframe width="600" height="371" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/>spreadsheet_id>/pubchart?oid=1220039972&amp;format=interactive"></iframe>
```

And it comes out looking great!

![image](https://user-images.githubusercontent.com/16181459/192843172-044f027f-d0bc-4347-9d77-c7cac379cf4c.png)

Obviously we might want to tweak the layout a bit, but this is a very easy thing to add. :)

**Comment by joshuacwnewton** [comment]
One thing we might want to do before publishing is to go through the Pathology and Tools charts and make sure they've been entered and parsed correctly. Some immediate thoughts:

* We use acronyms for pathology, but it might not be 100% clear what each of these is referring to, so we may want to swap e.g. HC for Healthy Control for clarity purposes, assuming it fits space-wise on the chart.
* Many of the entries for "tools used" is something like "sct_register_to_template, sct_dmri_compute_dti", but I don't think these are parsed as individual tools for the chart, so we get a lot of duplicate/overlapping one-off entries. We might want to figure out how to get Google Sheets to properly parse comma-delimited entries as individual counts for the chart.

**Comment by jcohenadad** [comment]
> We use acronyms for pathology, but it might not be 100% clear what each of these is referring to, so we may want to swap e.g. HC for Healthy Control for clarity purposes, assuming it fits space-wise on the chart.

good call. HC for Healthy Control, definitely. There seem to be space. The other accronyms are known by the community (and i don't think there is space, that's the bigger issue). eg: "spinal cord compress" is cropped, so we can simply say "Compression"

> Many of the entries for "tools used" are something like "sct_register_to_template, sct_dmri_compute_dti", but I don't think these are parsed as individual tools for the chart, meaning we get a lot of duplicate/overlapping one-off entries. So, we might want to figure out how to get Google Sheets to properly parse comma-delimited entries as individual counts for the chart.

👍 

> Also, a lot of "tools used" entries are tool names ("sct_X") while others are descriptors ("Segmentation", "CSA") so we might want to unify these, too.

👍 

as a first step, i would already publish this as is (better than nothing), and then we can work on these suggested improvements
